### PR TITLE
Covering edge cases for zero values in all reported losses

### PR DESCRIPTION
**Pull request by guilherme-brandao**
-> Covering edge cases where:
1) All reported inferer values are 0
2) All reported forecaster values are 0
...
Considering all properties of the Reputer Value Bundle. 

-> Critical fix where we start using the fTolerance from the topic instead of a mocked one. 

Ticket: https://linear.app/upshot/issue/ORA-1453/cover-edge-case-zero-values-in-all-reported-losses-values

### CODE REVIEW

**Review by relyt29 — APPROVED** [review]
approved pending tests pass (github actions are being wonky today)

File: x/emissions/module/rewards/worker_rewards.go [hunk 1]
```diff
@@ -99,7 +99,11 @@ func GetWorkersRewardFractions(
 		scores = append(scores, workerLastScoresDec)
 	}
 
-	rewardFractions, err := GetScoreFractions(latestWorkerScores, flatten(scores), pRewardSpread)
+	epsilon, err := k.GetParamsEpsilon(ctx)
```
Comment (kpeluso) [inline x/emissions/module/rewards/worker_rewards.go:102]: you could probs pass this into the function scope instead of this getter

**Review by kpeluso — APPROVED** [review]
